CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ New deprecation(s):
 ### Other
 
 - **General**: Refactor ScaledJob related methods to be located at scale_handler  ([#4781](https://github.com/kedacore/keda/issues/4781))
+- **General**: Add `serverAPIPath` option to Prometheus scaler if not found at `/api/v1/query`, which is the default (supporting Grafana Cloud hosted Prometheus)
```

Comment:
Could you open an issue and reference it here? 🙏

Reply:
https://github.com/kedacore/keda/issues/4819

Reply:
```suggestion
- **General**: Add `serverAPIPath` option to Prometheus scaler if not found at `/api/v1/query`, which is the default (supporting Grafana Cloud hosted Prometheus) ([#4819](https://github.com/kedacore/keda/issues/4819))
```